### PR TITLE
wc: move help strings to markdown file

### PR DESCRIPTION
**Pull request by schoki040**
#4368 

`wc --help` now outputs the following:

```
./target/debug/coreutils wc --help
Display newline, word, and byte counts for each FILE, and a total line if
more than one FILE is specified. With no FILE, or when FILE is -, read standard input.

Usage: ./target/debug/coreutils wc [OPTION]... [FILE]...

Arguments:
  [files]...

Options:
  -c, --bytes            print the byte counts
  -m, --chars            print the character counts
      --files0-from <F>  read input from the files specified by
                             NUL-terminated names in file F;
                             If F is - then read names from standard input
  -l, --lines            print the newline counts
  -L, --max-line-length  print the length of the longest line
  -w, --words            print the word counts
  -h, --help             Print help information
  -V, --version          Print version information
```

### CODE REVIEW

File: src/uu/wc/wc.md [hunk 1]
```diff
@@ -0,0 +1,9 @@
+# wc
+
+## Usage
```
Comment (tertsdiepraam) [inline src/uu/wc/wc.md:3]: This line shouldn't be necessary anymore

File: src/uu/wc/src/wc.rs [hunk 2]
```diff
@@ -98,9 +98,8 @@ impl Settings {
     }
 }
 
-static ABOUT: &str = "Display newline, word, and byte counts for each FILE, and a total line if
-more than one FILE is specified. With no FILE, or when FILE is -, read standard input.";
-const USAGE: &str = "{} [OPTION]... [FILE]...";
+static ABOUT: &str = help_about!("wc.md");
```
Comment (tertsdiepraam) [inline src/uu/wc/src/wc.rs:101]: Doesn't make much of a difference, but let's make them both `const` for consistency.

**Review by tertsdiepraam — COMMENTED** [review]
Thanks! I've approved the CI and left 2 small comments

File: src/uu/wc/src/wc.rs (hunk 2 above)
Comment (schoki040) [reply src/uu/wc/src/wc.rs:101]: Hi, no problem, I didnt want to tinker with original code. Coming right up.

File: src/uu/wc/wc.md (hunk 1 above)
Comment (schoki040) [reply src/uu/wc/wc.md:3]: Hi @tertsdiepraam, I'm sorry it shows me the first three lines. Which one is redundant?

File: src/uu/wc/src/wc.rs (hunk 2 above)
Comment (schoki040) [reply src/uu/wc/src/wc.rs]: Done

File: src/uu/wc/wc.md (hunk 1 above)
Comment (schoki040) [reply src/uu/wc/wc.md]: removed

File: src/uu/wc/wc.md [hunk 3]
```diff
@@ -0,0 +1,8 @@
+# wc
+
+```sh
```
Comment (sylvestre) [inline src/uu/wc/wc.md:3]:
```suggestion
```
```

**Pull request merged**